src/components/Authorization/LoginForm.tsx

```diff
@@ -61,6 +61,7 @@ export const LoginForm: React.FC<LoginFormProps> = ({
             id="email"
             type="text"
             placeholder="ukázka@email.com"
+            className="light"
```

Comment:
Don't name it className; that is reserved for CSS class names :D. Name it whatever else - variant f.e. It wont break anything, but it is unexpected API

Reply:
renamed className to variant